test/UnitTest/Microsoft.AspNet.OData.Test.Shared/TestCommon/Models/TestDataModels.cs

```diff
@@ -54,4 +57,26 @@ public class AutoExpandChoiceOrder
         public int Id { get; set; }
         public string Name { get; set; }
     }
+
+    public class NewCustomerUnmapped
```

Comment:
>NewCustomerUnmapped [](http://example.com/codeflow?start=17&length=19)

maybe add a [DataMember] attribute to a property that also has [IgnoreDataMember] and validate that property is serialized if [DataContract] is not applied to the type.